src/s2wasm.h

```diff
@@ -159,16 +159,23 @@ class S2WasmBuilder {
     return cashew::IString(str.c_str(), false);
   }
 
-  int32_t getInt() {
-    int32_t ret = 0;
+  uint32_t getInt() {
```

Comment:
You mean, it does the right thing while passing the negative values as unsigned (2s complement)?

Seems very confusing to me personally, as small negative numbers are very natural and common here.

What is the undefined behavior this counters? Perhaps there's another solution?